utils/src/utils/AzureResourceIdTelemetry.ts

```diff
@@ -0,0 +1,18 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { TelemetryTrustedValue } from "vscode";
+import { IActionContext } from "../..";
+
+export function setAzureResourceIdTelemetryProperties(context: IActionContext, resourceId: string): void {
+    if (isAzureResourceId(resourceId)) {
+        context.telemetry.properties.resourceId = new TelemetryTrustedValue(resourceId);
```

Comment:
In any given command scenario, is the assumption that we will only ever want to populate one resource id?

Reply:
Yep, this is only for actions against a specific resource. So like, if I deploy to my app, etc.